src/sagemaker/processing.py

```diff
@@ -797,15 +796,21 @@ def from_processing_name(cls, sagemaker_session, processing_job_name):
         if job_desc.get("ProcessingOutputConfig") and job_desc["ProcessingOutputConfig"].get(
             "Outputs"
         ):
-            outputs = [
-                ProcessingOutput(
-                    source=processing_output["S3Output"]["LocalPath"],
-                    destination=processing_output["S3Output"]["S3Uri"],
-                    output_name=processing_output["OutputName"],
+            outputs = []
+            for processing_output_dict in job_desc["ProcessingOutputConfig"]["Outputs"]:
+                processing_output = ProcessingOutput(
+                    output_name=processing_output_dict["OutputName"],
+                    app_managed=processing_output_dict.get("AppManaged") or False,
```

Comment:
If `AppManaged` key is present, is the value ever `None`? If not, specify default `False` on the `get` call?

```suggestion
                    app_managed=processing_output_dict.get("AppManaged", False),
```

Reply:
So .get("AppManaged") is None for previous processing jobs.

Reply:
Done.